webrtc-sys/src/audio_track.cpp

```diff
@@ -163,15 +163,15 @@ AudioTrackSource::InternalSource::InternalSource(
 
         if (buffer_.size() >= samples10ms) {
           for (auto sink : sinks_)
-            sink->OnData(buffer_.data(), sizeof(int16_t), sample_rate_,
+            sink->OnData(buffer_.data(), sizeof(int16_t) * 8, sample_rate_,
```

Comment:
ah blah, thx

Reply:
funny that it works without it

Reply:
yeah, assert is only triggered in debug mode.